mods/ctf_pvp_engine/ctf_chat/init.lua

```diff
@@ -159,64 +159,62 @@ minetest.register_chatcommand("team", {
 })
 
 minetest.register_chatcommand("access", {
-	description = "Open the team console, or run team command (see /team help)",
+	description = "Run access command (see /access help)",
```

Comment:
Can still be improved. A better description would be to briefly explain what the access command actually is.

Reply:
What would you saw that will make a better description?